### PR TITLE
[RW-9892][risk=low] Update Cromwell snippet to replace cromshell-beta with Cromshell

### PR DESCRIPTION
**Pull request by yonghaoy**
We are about to upgrade our docker image that has Cromshell as alias

https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-aou/Dockerfile#L266

\<your comments for this pull request go here\>

Unfortunately we don't have automated testing configured for the code in this
repository yet so we set up this checklist as an *automatic reminder*:

- [ ] Ensure that the smoke tests pass using the current (or upcoming) CDR
- [ ] Update documentation relevant to this pull request

Questions? See [CONTRIBUTING.md](https://github.com/all-of-us/workbench-snippets/blob/master/CONTRIBUTING.md)
or file an issue so that we can get it documented!

### CODE REVIEW

File: py/py_cromwell_setup.py [hunk 1]
```diff
@@ -55,8 +55,8 @@ def get_app_details(env, app_name):
 
 # Checks that cromshell is installed. Otherwise raises an error.
 def validate_cromshell():
-    if validate_cromshell_beta():
-        print("Found cromshell-beta, please use cromshell-beta")
+    if validate_cromshell():
+        print("Found cromshell, please use cromshell")
     elif validate_cromshell_alpha():
```
Comment (NehaBroad) [inline py/py_cromwell_setup.py:60]: Do we still need to check for alpha?

**Comment by NehaBroad** [comment]
Will you be opening corresponding workbench branch for this change right now or after docker changes?

**Comment by yonghaoy** [comment]
> Will you be opening corresponding workbench branch for this change right now or after docker changes?

https://github.com/all-of-us/workbench/pull/7695 just have this pr up

File: py/py_cromwell_setup.py (hunk 1 above)
Comment (NehaBroad) [reply py/py_cromwell_setup.py:60]: we have two function with same name validate_cromshell

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**